src/main/kotlin/com/linchpino/ai/model/AIServiceName.kt

```diff
@@ -0,0 +1,32 @@
+package com.linchpino.ai.model
+
+import com.linchpino.ai.service.impl.ChatGPTServiceImpl
+import com.linchpino.ai.service.impl.GeminiServiceImpl
+
+enum class AIServiceName(val label: String, @JvmField val componentName: String, private val isDefault: Boolean) {
```

Comment:
this enum is redundant. You can reach the same functionality with one line of code

Reply:
same https://github.com/linchpino/linchpino-core/pull/147#discussion_r1808183204